examples/connext_secure/certificate_revocation_list/c++11/USER_QOS_PROFILES.xml

```diff
@@ -52,7 +52,7 @@ to use the software. -->
               </element>
               <element>
                 <name>com.rti.serv.secure.files_poll_interval</name>
-                <value>5000</value>
+                <value>5</value>

```

Comment:
Is this change related to your PR at all?

Reply:
No it is not. I opened a different PR in April for this change. I discovered this when Support was doing "Training as One" to learn about new features and this example was broken. https://github.com/rticommunity/rticonnextdds-examples/pull/684